create/create.go

```diff
@@ -362,12 +362,21 @@ func formatContent(targetPath string, content []byte) ([]byte, error) {
 	return content, nil
 }
 
-func formatJSON(bytes []byte) ([]byte, error) {
+func formatJSON(input []byte) ([]byte, error) {
 	var result map[string]interface{}
-	if err := json.Unmarshal(bytes, &result); err != nil {
+	if err := json.Unmarshal(input, &result); err != nil {
 		return nil, err
 	}
-	return json.MarshalIndent(result, "", "  ")
+
+	var output bytes.Buffer
+	encoder := json.NewEncoder(&output)
+	encoder.SetEscapeHTML(false)
```

Comment:
Turns out [`json.Marshall`](https://pkg.go.dev/encoding/json#Marshal) and `json.MarshallIndent` perform escaping of `<` and `>` and a few other characters.

> String values encode as JSON strings coerced to valid UTF-8, replacing invalid bytes with the Unicode replacement rune. So that the JSON will be safe to embed inside HTML <script> tags, the string is encoded using HTMLEscape, which replaces "<", ">", "&", U+2028, and U+2029 are escaped to "\u003c","\u003e", "\u0026", "\u2028", and "\u2029". This replacement can be disabled when using an Encoder, by calling SetEscapeHTML(false).

There is no option to disable it other than instantiating an `Encoder` manually and configuring the instance appropriately.